src/app/public/modules/description-list/types/description-list-orientation.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Specifies the orientation for the description list. The default `vertical` orientation
+ * stacks term-description pairs in a vertical list, while the `horizontal` orientation displays
+ * term-description pairs side by side in a horizontal list. This only applies to description lists
```

Comment:
I had the same thought on this... Glad you brought it up. I'd also +1 having 3 modes instead of an `orientation`. Once we get the naming sorted out (see other comment), I will make the change.